grpc-bridge/src/main/java/org/jboss/resteasy/grpc/util/IteratorEnumeration.java

```diff
@@ -0,0 +1,29 @@
+package org.jboss.resteasy.grpc.util;
+
+import java.util.Enumeration;
+import java.util.Iterator;
+
+/**
+ * Wrapper to convert an iterator to an enumeration
+ *
+ * @author Stuart Douglas (from undertow)
+ */
+public class IteratorEnumeration<T> implements Enumeration<T> {
```

Comment:
This shouldn't be required. There is a `Collections.enumeration(Collection)` you could use instead.

Reply:
I just copied Stuart's code. I'll have to go back and see if there was a reason I copied the code instead of just using the classes from undertow.

Reply:
I've stripped out the classes I copied from undertow and call the original classes directly.